tests/integration/test_api/test_statistics/test_agent_statistics_format.py

```diff
@@ -0,0 +1,89 @@
+import jsonschema
+import os
+import pytest
+
+from wazuh_testing import event_monitor as evm
+from wazuh_testing.api import make_api_call, get_api_details_dict
+from wazuh_testing.tools import LOG_FILE_PATH, CLIENT_KEYS_PATH
+from wazuh_testing.tools.configuration import get_test_cases_data
+from wazuh_testing.tools.file import read_json_file, read_file
+
+pytestmark = [pytest.mark.agent]
```

Comment:
We are checking only the response format. It is necessary to do this in a `wazuh-agent`? Or we can mock an agent and checks it directly in the `wazuh-manager`?

Reply:
Done in d890695